Form-Controls/index.html

```diff
@@ -13,15 +13,43 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label for="name">Name</label>
+          <input type="text" name="name" id="name" placeholder="YourName" required>
+        </div>
+        <br>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" placeholder="YourEmail" required>
+        </div>
+        <br>
+        <div>
+          <label for="tshirt_colour">T-shirt Colour</label>
```

Comment:
I recommend getting in the habit of using kebab-case when naming your HTML attributes.
Instead of the `<select name="T-shirt_colour" id="tshirt_colour" required>`, use kebab-case for the name and id, like this: `<select name="tshirt-colour" id="tshirt-colour" required>`. Notice how I use all lowercase and a hyphen in between words? This makes your code easier to read.  
<br>
You can read more here: https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case
and here: https://dev.to/codepapi/naming-conventions-and-style-guides-in-programming-59a9